addons/web/models/models.py

```diff
@@ -12,6 +12,7 @@
 from odoo.exceptions import UserError
 from collections import defaultdict
 
+SEARCH_PANEL_ERROR = {'error_msg': _("Too many items to display."), }
```

Comment:
You cannot call the translation function `_` in the global namespace of a module.
Move that thing into a method.

Reply:
ok